java/src/test/java/com/saucelabs/simplesauce/BaseConfigurationTest.java

```diff
@@ -1,21 +1,18 @@
 package com.saucelabs.simplesauce;
 
-import com.saucelabs.simplesauce.interfaces.EnvironmentManager;
-import com.saucelabs.simplesauce.interfaces.SauceRemoteDriver;
 import org.junit.Before;
 
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 class BaseConfigurationTest {
-    public SauceSession sauce;
-    protected SauceRemoteDriver dummyRemoteDriver;
-    protected EnvironmentManager dummyEnvironmentManager;
-    protected SauceOptions sauceOptions;
+    SauceSession sauce;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4594401)